### DISCUSSION TITLE
"Onvif Device Test Tool"download?

### DISCUSSION THREAD

**Discussion by wkun0702** [onvif/testspecs#7] [Q&A] [answered]
hello,
Where can I download the Onvif device test tool?So far I have not found the download entry?
Please give me some guidance. Thank you

**Comment by Johan-Svensk** [comment, marked as answer]
Hi!
The test tools can be found in the [ONVIF Member Portal](https://www.onvif.org/member-portal) under "Test Tool Download".

Regards,
Johan

**Reply by baldvin137** [reply]
Additional information: The ONVIF test tools are available to ONVIF member companies only. Access to the ONVIF Member Portal is administered by the individual member companies. If you do not have login access to the ONVIF Member Portal, please contact the ONVIF administrator at your own company.
